pkg/signature/signerverifier_options.go

```diff
@@ -0,0 +1,70 @@
+//
```

Comment:
can you move this file into `options` subdir please?

Reply:
Moving this into `options` mean changing the package to `options`. Then the `makeSignerVerifierOpts` needs to be made public, as well as the `loadOpts` structure, which would make things more rigid and harder to modify.

Is this what we want? Am I missing a nice trick to avoid this?
`LoadSignerWithOpts` (and the other functions) are in `signature` package and they should be able to easily construct the options object.

Reply:
why can't you define an interfaces in `signature/options.go` and then put the implementation in `signature/options/loadoptions.go`?

Reply:
Ok I've changed the way I implement the option pattern and now it should be fine!